### ISSUE TITLE
请问cookie是 jwt 哪个token吗

### ISSUE THREAD

**Issue by lizhemin15** [yihong0618/SunoSongsCreator#6]
填了后报错 Exception: Failed to get session id

**Comment by yihong0618** [comment]
不是 F12 XHR，https://clerk.suno.ai/v1/client?_clerk_js_version=4.70.5 这个网址里面的 cookie

**Comment by lizhemin15** [comment]
> 不是 F12 XHR，https://clerk.suno.ai/v1/client?_clerk_js_version=4.70.5 这个网址里面的 cookie

访问了出来的这个，是整体复制吗，还是哪个字段 
![image](https://github.com/yihong0618/SunoSongsCreator/assets/57975421/1c1fdf40-f874-4511-b211-157ffd67c724)

**Comment by yihong0618** [comment]
不是这个是 header 里的 cookie 整个复制

**Comment by lizhemin15** [comment]
> 不是这个是 header 里的 cookie 整个复制

懂了，谢谢佬，后续的人如果有相同的问题的，可按如下步骤操作：
1. 浏览器进入大佬给定网址
2. 使用浏览器插件 Cookie-Editor点Export 为Header String
3. 得到形如"__client_uat=1711375871;__cf_bm=OrPl3UdpYAYLiBRJgN9Aol6Utx6zC2LRY6C7q517xxxx"的就是需要的cookie了

大佬，关于这部分建议在说明中再稍微说说？我这种小白上手就还有一定门槛

**Comment by yihong0618** [comment]
> > 不是这个是 header 里的 cookie 整个复制
> 
> 
> 
> 懂了，谢谢佬，后续的人如果有相同的问题的，可按如下步骤操作：
> 
> 1. 浏览器进入大佬给定网址
> 
> 2. 使用浏览器插件 Cookie-Editor点Export 为Header String
> 
> 3. 得到形如"__client_uat=1711375871;__cf_bm=OrPl3UdpYAYLiBRJgN9Aol6Utx6zC2LRY6C7q517xxxx"的就是需要的cookie了
> 
> 
> 
> 大佬，关于这部分建议在说明中再稍微说说？我这种小白上手就还有一定门槛

好

**Comment by BuwadsMod** [comment]
help 


my:1 Third-party cookie will be blocked. Learn more in the Issues tab.
my:1 Third-party cookie will be blocked. Learn more in the Issues tab.
my:1 Third-party cookie will be blocked. Learn more in the Issues tab.
music/7039f91479d043ae92662a238cd2c4c1:1 Third-party cookie will be blocked. Learn more in the Issues tab.
music/7039f91479d043ae92662a238cd2c4c1:1 Third-party cookie will be blocked. Learn more in the Issues tab.
music/7039f91479d043ae92662a238cd2c4c1:1 Third-party cookie will be blocked. Learn more in the Issues tab.
music/7039f91479d043ae92662a238cd2c4c1:1 Third-party cookie will be blocked. Learn more in the Issues tab.
4986-5707beec9baa2482.js:1 There are no focusable elements inside the <FocusTrap />

**Comment by BuwadsMod** [comment]
i cant find it Server:
cloudflare
Set-Cookie:
__Secure-next-auth.session-token=eyJhbGciOiJkaXIiLCJlbmMiOiJBMjU2R0NNIn0..W8sjpCR52_VMTctT.OMFbIzWZ47S-WuY3TLvflzFiorNPJv3kJfmJqYkG6A8wyOw9sSTb4sZSNTJvEu2GpXmblF19RYQ7XZUwVSB0McVyNx5V6saPlJMqOPoufjmAjNRPKagimJBlQqRnIZwKmmpETjO1VzoD2LWrWK74L3hWM_YgRP0lHJt5Zx9fNhsf6Dwcq_MVLMxXZD9Vb0XvsINpLiW3YECifVOitrZqjCSZgaExu5ixpmKdj0wcRVMJx_yuMDkk3nZvBR9CZg4rqk6gfJzDEyYqum6RBsYgVOieg-TBk_4QD93-Gl7SF03EK66ll4Gl_e77m6OMGU9PEUkfJFcKrPvI_f4Vj5uhMw.yUh-_xL-uY7jeaZfrffRVg; Path=/; Expires=Tue, 30 Apr 2024 07:21:36 GMT; HttpOnly; Secure; SameSite=Lax
Strict-Transport-Security:
max-age=63072000
Vary:
RSC, Next-Router-State-Tree, Next-Router-Prefetch, Next-Url
X-Matched-Path:
/[locale]/api/auth/[...nextauth]
X-Vercel-Cache:
MISS
X-Vercel-Execution-Region:
iad1
X-Vercel-Id:
sin1::iad1::6gbcq-1711869696665-534afca6737e

**Comment by iSte94** [comment]
it's the same with the new version?
![image](https://github.com/yihong0618/SunoSongsCreator/assets/108139714/f8f56a8c-48b2-43d0-9a01-525784917b42)

**Comment by yihong0618** [comment]
yes。

**Comment by BuwadsMod** [comment]
> it's the same with the new version? ![image](https://private-user-images.githubusercontent.com/108139714/318408361-f8f56a8c-48b2-43d0-9a01-525784917b42.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTIwMjAxNTIsIm5iZiI6MTcxMjAxOTg1MiwicGF0aCI6Ii8xMDgxMzk3MTQvMzE4NDA4MzYxLWY4ZjU2YThjLTQ4YjItNDNkMC05YTAxLTUyNTc4NDkxN2I0Mi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDAyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQwMlQwMTA0MTJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04YTljZDY1MzE1ZTRmODk0ZDNjZWUwZjBkYWZhYWJiMzI5OWNjNDFiZmQ2MWU3NWQ3YWE4OTlhNDEzMDUyNTcwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.XpiGIVs5H26zdoQVm8nAJu4WOiAbR8M-g188aYBgOu8)

how did you manage to get this sir

**Comment by BuwadsMod** [comment]
what do i put here in client_uat 

![image](https://github.com/yihong0618/SunoSongsCreator/assets/164748997/5d7fab27-f6e8-40a6-a4cc-7c26e57fa8e3)

**Comment by yihong0618** [comment]
you need copy all
得到形如"__client_uat=1711375871;__cf_bm=OrPl3UdpYAYLiBRJgN9Aol6Utx6zC2LRY6C7q517xxxx"的就是需要的cookie了

**Comment by BuwadsMod** [comment]
> you need copy all to get the cookie that looks like "__client_uat=1711375871;__cf_bm=OrPl3UdpYAYLiBRJgN9Aol6Utx6zC2LRY6C7q517xxxx"

boss i cant find that kind of cookie its different this is mine 


_ga =GA1.1.656884068.1711868496

is this it ?

**Comment by enteryfx** [comment]
> you need copy all 得到形如"__client_uat=1711375871;__cf_bm=OrPl3UdpYAYLiBRJgN9Aol6Utx6zC2LRY6C7q517xxxx"的就是需要的cookie了

你好，我尝试访问您给出的url 显示
![8ad059f2282f6d1b321c8f434a7c245](https://github.com/yihong0618/SunoSongsCreator/assets/100504675/798c1cf4-01a9-4f36-b9c5-f463b0dd73d7)
you have been blocked 
可以解答一下这是为什么吗

**Comment by enteryfx** [comment]
> 应该是被风控封号了。

但是用别的电脑，从来没有登录过suno，也是这样，也是风控了嘛。。。

**Comment by yihong0618** [comment]
不知道。。。

**Comment by enteryfx** [comment]
> 不知道。。。

哈哈好吧，我再尝试一下
